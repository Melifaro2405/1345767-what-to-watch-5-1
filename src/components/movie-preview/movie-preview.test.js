import React from 'react';
import renderer from 'react-test-renderer';
import MoviePreview from './movie-preview';

it(`Should MoviePreview render correctly`, () => {
  const tree = renderer
    .create(
        <MoviePreview playVideoSrc={`test`} src={`test`} />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
