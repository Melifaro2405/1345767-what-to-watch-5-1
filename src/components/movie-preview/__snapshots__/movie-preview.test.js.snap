// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MoviePreview render correctly 1`] = `
<video
  autoPlay={true}
  className="player__video"
  muted={true}
  poster="test"
  src="test"
/>
`;
